src/dotnet/APIView/APIViewWeb/Pages/Assemblies/Review.cshtml

```diff
@@ -482,7 +482,12 @@
                     var diffRevisionType = (Model.ReviewContent.APIRevisionsGrouped.ContainsKey("Automatic")) ? "Automatic" : Model.ReviewContent.APIRevisionsGrouped.Keys.First();
                     @if (Model.DiffRevisionId != null)
```

Comment:
Should this DiffRevisionId condition be changed like you did below?

Reply:
Yes . I will change that too. Thanks